deploy/charts/monitoring/dashboards/central.json

```
@@ -117,18 +118,19 @@
             "uid": "PBFA97CFB590B2093"
           },
           "exemplar": true,
-          "expr": "increase(process_cpu_seconds_total{instance=~\"$Instance\",job!=\"kubernetes-pods\"}[1m])",
+          "expr": "sum by(pod)(\n  increase(container_cpu_usage_seconds_total{namespace=\"stackrox\", container=~\"(collector|sensor|compliance|admission-control|node-inventory|central|central-db|scanner|scanner-db)\"}[1m])\n)",
```

```suggestion
          "expr": "sum by(pod)(\n  rate(container_cpu_usage_seconds_total{namespace=\"stackrox\", container=~\"(collector|sensor|compliance|admission-control|node-inventory|central|central-db|scanner|scanner-db)\"}[1m])\n)",
```
Let's use `rate` instead of `increase` here. That should show CPU utilization, as in a value of `1` means that the CPU is fully utilized. I'd also recommend to set the unit in Grafana to percent, such that it would become `100%` instead of `1`. I think that's more intuitive than CPU seconds.

---

Done